crates/findex/src/pyo3/py_api.rs

```diff
@@ -536,3 +518,213 @@ impl InternalFindex {
         Ok(())
     }
 }
+
+#[pyclass]
+pub struct FindexCloud(FindexCloudRust);
+
+#[pymethods]
+impl FindexCloud {
+    #[new]
+    pub fn new(token: &str, base_url: Option<String>) -> PyResult<Self> {
+        Ok(Self(pyo3_unwrap!(
+            FindexCloudRust::new(token, base_url),
+            "error reading token"
+        )))
+    }
+
+    /// Upserts the given relations between `IndexedValue` and `Keyword` into
+    /// Findex tables. After upserting, any search for a `Word` given in the
+    /// aforementioned relations will result in finding (at least) the
+    /// corresponding `IndexedValue`.
+    ///
+    /// Parameters
```

Comment:
Is there a reason for not using `# Parameters`?

Reply:
I used the same formatting as the previous functions